internal/osbuildexecutor/runner-impl-aws-ec2.go

```diff
@@ -53,52 +95,133 @@ func (ec2e *awsEC2Executor) RunOSBuild(manifest []byte, store, outputDirectory s
 		}
 	}()
 
-	logrus.Info("Spinning up jobsite manager")
-	args := []string{
-		"--builder-host",
-		*si.Instance.PrivateIpAddress,
-		"--store",
-		store,
+	ctx, cancel := context.WithTimeout(context.Background(), time.Minute*10)
+	defer cancel()
+	if !waitForSI(ctx, *si.Instance.PrivateIpAddress) {
+		return nil, err
 	}
+	controlFile, err := os.Open(filepath.Join("/var/tmp/osbuild-composer", "control.json"))
+	if err != nil {
+		return nil, err
+	}
+	defer os.Remove(controlFile.Name())
+	_, _ = controlFile.Write([]byte(fmt.Sprintf("{\"exports\": [\"%s\"]}", exports[0])))
+	_ = controlFile.Sync()
 
-	for _, exp := range exports {
-		args = append(args, "--export", exp)
+	manifestFile, err := os.Open(filepath.Join("/var/tmp/osbuild-composer", "manifest.json"))
+	if err != nil {
+		return nil, err
 	}
-	for _, exp := range exportPaths {
-		args = append(args, "--export-file", exp)
+	defer os.Remove(manifestFile.Name())
+	_, _ = manifestFile.Write(manifest)
+	_ = manifestFile.Sync()
+
+	cmd := exec.Command("tar",
+		"--strip-components=3",
+		"-cf",
+		filepath.Join("/var/tmp/osbuild-composer", "input.tar"),
+		filepath.Join("/var/tmp/osbuild-composer", "control.json"),
+		filepath.Join("/var/tmp/osbuild-composer", "manifest.json"),
+	)
+	if err = cmd.Run(); err != nil {
```

Comment:
Maybe worth using `cmd.Output()` or `cmd.CombinedOutput()` here and returning it when an error happens? Something like:
```go
	if output, err := cmd.CombinedOutput(); err != nil {
		logrus.Errorf("Unable to create input tar: %v output:\n%s", err, output)
		return nil, err
	}
```
there is often useful context in the output like "disk-full" and the standard behavior of exec.Command.Run() is to just include the error code which is often not very helpful